### PR TITLE
website-updation

### LINKED ISSUE #82: Website updation
### Landing page

- [x] remove what'snew ke niche jo everything hai from the navbar 

image

### About us https://www.srmkzilla.net/about

- [x] get in touch with edi and events assoc and leads and update this pic

image

- [ ] Update other pictures and content, make it more recent

### our services 

- [ ] Update the testimonials and add new ones

- [x] Update the let's team up and add a cal.com link there instead 

- [ ] Update this 
image

### PR DESCRIPTION
**Pull request by utkarshini14**
# Description

Updated the following:

### Landing page

- [x] remove what's new ke niche jo everything hai from the navbar 

Screenshot 2023-08-03 000213

### About us https://www.srmkzilla.net/about

- [x] get in touch with edi and events assoc and leads and update this pic

image

- [x] Update other pictures and content, make it more recent

### our services 

- [ ] ~~Update the testimonials and add new ones~~

- [x] Update the let's team up and add a cal.com link there instead 

- [x] Update the content of mozofest and mozohack
image

Fixes #82 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: shared/components/services/collaborate.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const Collaborate = () => {
             <span className="font-bold text-orange600">Together</span> we can
             create something magnificent.
           </p>
-          <a href="mailto:hello@srmkzilla.net" rel="noopener noreferrer">
+          <a href="https://cal.com/srmkzilla/hello" rel="noopener noreferrer">
```
Comment (shawshankkumar) [inline shared/components/services/collaborate.tsx:15]: target="_blank"

**Review by shawshankkumar — CHANGES_REQUESTED** [review]

File: pages/about.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const AboutUs = ({ timelineData }) => {
         <div className="md:h-screen h-auto items-center flex justify-center">
           <div className="md:h-3/4 h-full sm:w-10/12 w-full md:mb-16 mb-8 rounded-none md:rounded-xl overflow-hidden">
             <img
-              src="/images/mozofest.png"
```
Comment (shawshankkumar) [inline pages/about.tsx:25]: remove mozofest.png from public if it is not being used elsewhere

**Review by shawshankkumar — CHANGES_REQUESTED** [review]

**Comment by shawshankkumar** [comment]
image


 update photos here and content too

**Comment by shawshankkumar** [comment]
issue galat linked hai, # and the number has to be together without any space

**Comment by shawshankkumar** [comment]
#83  like this

**Comment by shawshankkumar** [comment]
@utkarshini14 get these done by tonight, mark it ready for review, get it reviewed by me and one of the leads and let's get this merged.

**Review by shawshankkumar — APPROVED** [review]
LGTM! 🚀

**Review by sm-sami — APPROVED** [review]
LGTM! 🚀

**Pull request merged**